pyserini/encode/_splade.py

```diff
@@ -33,3 +36,13 @@ def _output_to_weight_dicts(self, batch_aggregated_logits):
             d = {self.reverse_voc[k]: float(v) for k, v in zip(list(col), list(weights))}
             to_return.append(d)
         return to_return
+
+    def _get_encoded_query_token_wight_dicts(self, tok_weights):
```

Comment:
Hi @AileenLin - if we implement the quantization and API changes we discussed on the Java end, we wouldn't need this on the Python end, right?